Modules/Authentication/Multi/MultiAuthenticationConcern.cs

```diff
@@ -0,0 +1,83 @@
+﻿using GenHTTP.Api.Content;
+using GenHTTP.Api.Protocol;
+
+namespace GenHTTP.Modules.Authentication.Multi;
+
+public sealed class MultiAuthenticationConcern : IConcern
+{
+    #region Get-/Setters
+
+    public IHandler Content { get; }
+
+    private readonly IConcern[] _delegatingConcerns;
+
+    #endregion
+
+    #region Initialization
+
+    public MultiAuthenticationConcern(IHandler content, IConcern[] delegatingConcerns)
+    {
+        Content = content;
+        _delegatingConcerns = delegatingConcerns ?? [];
```

Comment:
With nullable enabled the `?? []` part cannot happen, no?

Reply:
Hmm I'm using nullable disabled in all my project as I didn't get used to this feature yet. So I'm used to a bit of a defensive coding style. So please bear with me. 😆  I'll review and update.

There is probably one question to you - developers can still use nullable disabled in their projects, while they reference genhttp that uses nullable enabled. This in practice means, that they can pass null to genhttp methods where null is not expected. This compiles and runs just fine, and you can still get nullreferenceexception inside genhttp. So the question is - do you want to be defensive for this scenario or not? (I guess probably not as that's how the rest of genhttp is written).

Reply:
Yes, I think I agree on this one - but only for the public API surface - so the main entry point should be `MultiAuthentication` and builder and not the concern (which _could_ be `internal` theoretically).

I feel nullable support is kind of messy in C# - if I enable it, the CLR should automatically check for nulls and gate my non-nullable methods instead of just passings down `null` values. Seems to be implemented as a language feature and not a runtime CLR feature.

Nevertheless I think this should be decided on project level (see #576) rather than on PR level, so I would prefer removing the null checks for now.